assets/scss/base/_icons.scss

```diff
@@ -55,3 +55,19 @@
     background-size: cover;
   }
 }
+
+.govuk-verify {
+  background: {
```

Comment:
Interesting way of writing a style in sass - looks clearer, although feels unconventional